content/connect/_index.html

```diff
@@ -0,0 +1,93 @@
+---
+title: "Connect"
+date: 2020-02-03T22:26:21+11:00
+draft: false
+---
+
+<div class="columns is-centered">
+    <div class="column is-8">
+        <div class="section">
+            <h1 class="title is-2">Community</h1>
+            <div class="content">
+                <p>
+                    UQCS prides itself on promoting and facilitating a community of students
+                    engaged with software and technology. Join us on our community slack to
+                    chat about university, your latest side project or newest hobby! Follow
+                    the society Facebook page to stay up to date on the latest events and
+                    announcements. Finally, work on your skills by contributing to our open
+                    source projects.
+                </p>
+                <nav class="level">
+                    <div class="level-item has-text-centered">
+                        <div>
+                            <a href="https://slack.uqcs.org.au">
+                            <p class="heading">Chat</p>
+                            <p class="title">Slack</p>
+                                <span class="icon has-text-dark">
+                                    <i class="fab fa-3x fa-slack"></i>
+                                </span>
+                            </a>
+                        </div>
+                    </div>
+                    <div class="level-item has-text-centered">
+                        <div>
+                            <a href="https://www.facebook.com/uqcomputing">
+                            <p class="heading">Engage</p>
```

Comment:
This should be indented, as should the contents in the other <a> tags.

Reply:
Really should add a linter action 🤔